source/BoardingPanel.cpp

```diff
@@ -645,13 +648,17 @@ void BoardingPanel::Plunder::Take(int count)
 // Update the text to reflect a change in the item count.
 void BoardingPanel::Plunder::UpdateStrings()
 {
+
 	double mass = UnitMass();
 	if(count == 1)
 		size = Format::Number(mass);
 	else
 		size = to_string(count) + " x " + Format::Number(mass);
 
-	value = Format::Credits(unitValue * count);
+	if(!knownToPlayer)
```

Comment:
Switch the condition around, i.e. check for `if(knownToPlayer)`

Reply:
will take a look